team_picker/routes/teams.js

```diff
@@ -0,0 +1,94 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../db/client");
+
+router.get('/', (req, res) => {
+  res.render('main'); 
+});
+
+router.get("/new", (req, res) => {
+  res.render("new");
+});
+
+// Enter new cohort
+router.post("/new", (req, res) => {
+  console.log("Saving data to database from new teams")
+  knex("cohorts")
+    .insert({
+      name: req.body.name,
+      logoUrl: req.body.logoUrl,
+      members: req.body.members
+    })
+    .returning("*")
+    .then(cohorts => {
+      res.render('show', { cohort: cohorts[0], pickerMethod: undefined, quantity: 0 });
+    });
+});
+
+// List of all the cohorts
+router.get('/cohorts', (req, res) => {
+  knex("cohorts")
+    .orderBy("name")
+    .then(cohorts => {
+      res.render("index", { cohorts });
+    });
+});
+
+// Display a specific cohort and team picker
+router.get('/teams/:id', (req, res) => {
```

Comment:
You normally want your "related" URLs to be consistent, so they should either _all_ be `/cohorts` or they should all be `/teams`